docs/mochitests.md

```diff
@@ -34,6 +40,42 @@ reflected in the new firefox directory.
 * `yarn mochi -- --jsdebugger` opens a browser toolbox
 * `yarn mochi browser_dbg-editor-highlight` runs just one test
 
+## Writing Tests
+
+Here are a few tips for writing mochitests:
+
+* There are lots of great helper methods in [head]
+* Try to write async user actions that involve a user action like clicking a button or typing a key press followed by a redux action to listen for. For example, the user step in action involvesthe user clicking the step in button followed by the "stepIn" action firing.
```

Comment:
yeah - we should revisit this section the next time we come in here. I think it could be better factored :)